database/sql/V075__add_call_data_to_quotes_and_order_quotes.sql

```diff
@@ -0,0 +1,35 @@
+-- Step 1: Add a new column to the quotes table
+ALTER TABLE quotes 
+    ADD COLUMN verified boolean;
+
+-- Step 2: Add two new columns to the order_quotes table
+ALTER TABLE order_quotes 
+    ADD COLUMN verified boolean;
+
+-- Step 3: Create table with quote interactions
+CREATE TABLE quotes_interactions (
+    quote_id bigint NOT NULL,
+    index int NOT NULL,
+    target bytea NOT NULL,
+    value numeric(78,0) NOT NULL,
+    call_data bytea,
+
+    PRIMARY KEY (quote_id, index)
+);
+
+-- Get a specific quote's interactions.
+CREATE INDEX quote_id_interactions ON quotes_interactions USING HASH (quote_id);
+
+-- Step 3: Create table with quote interactions
+CREATE TABLE order_quotes_interactions (
+    order_uid bytea NOT NULL,
+    index int NOT NULL,
+    target bytea NOT NULL,
+    value numeric(78,0) NOT NULL,
+    call_data bytea,
+
+    PRIMARY KEY (order_uid, index)
+);
+
+-- Get a specific order's interactions.
+CREATE INDEX order_uid_interactions ON order_quotes_interactions USING HASH (order_uid);
```

Comment:
I haven't dived into the reasoning behind these tables, but the naming seems inconsistent.

```suggestion
CREATE TABLE quote_interactions (
    quote_id bigint NOT NULL,
    index int NOT NULL,
    target bytea NOT NULL,
    value numeric(78,0) NOT NULL,
    call_data bytea,

    PRIMARY KEY (quote_id, index)
);

-- Get a specific quote's interactions.
CREATE INDEX quote_id_interactions ON quote_interactions USING HASH (quote_id);

-- Step 3: Create table with quote interactions
CREATE TABLE order_quote_interactions (
    order_uid bytea NOT NULL,
    index int NOT NULL,
    target bytea NOT NULL,
    value numeric(78,0) NOT NULL,
    call_data bytea,

    PRIMARY KEY (order_uid, index)
);

-- Get a specific order's interactions.
CREATE INDEX order_uid_interactions ON order_quote_interactions USING HASH (order_uid);
```

Reply:
I was just following existing table names: we have `quotes` so created `quotes_interactions`, same for existing `order_quotes` I choose `order_quotes_interactions`.

Reply:
Following this logic, if we have `orders` table, then we should have `orders_quotes` instead of `order_quotes`, right? It doesn't align well with common database naming conventions.

Reply:
You are right, the current database tables could be named better, but for a now we have following names of tables:
| table name |
|-----------------|
| order_events    |
| order_execution |
| order_quotes    |
| orders          |

new table is related to `order_quotes` table so I prefer to name it just adding a suffix: `order_quotes_interactions`. I think using `orders_quotes_interactions` will suggest this is relevant to `orders` table which is not.

Similar for quotes, for a now we have only `quotes` table so adding a suffix we get: `quotes_interactions`.

Reply:
> the current database tables could be named better

I didn't mean that :) My suggestion is to follow common DB naming conventions. 
Let's use singular nouns to indicate each row relates to one `order_quote` and one `order`, not to many. The plural form can be ambiguous, as it might suggest multiple `order_quotes` are being interacted with in each row.